src/lib/dom.js

```diff
@@ -237,21 +237,24 @@ exports.scrollbarWidth = function(document) {
 
     outer.appendChild(inner);
 
-    var body = document.documentElement;
-    body.appendChild(outer);
+    var body = (doc && doc.documentElement) || (document && document.documentElement);
+    if (body) {
```

Comment:
it is better to do `if (!body) return 16;`

Reply:
Done!